public-api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -91,34 +90,28 @@ select 0, 5 as analysis_id,  CAST(ETHNICITY_CONCEPT_ID AS STRING) as stratum_1,
 from \`${BQ_PROJECT}.${BQ_DATASET}.person\`
 group by ETHNICITY_CONCEPT_ID"
 
-# 6	Number of persons by age decile 18-29
+# 6 Number of person by age decile
 echo "Getting age decile count"
 bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 "insert into \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.achilles_results\` (id, analysis_id, stratum_1, count_value,source_count_value)
-select 0, 6 as analysis_id,  '2' as stratum_2,
-COUNT(distinct person_id) as count_value, 0 as source_count_value
+with person_age as
+(select person_id,
+case when extract(month from current_date()) < (month_of_birth) and extract(day from current_date()) < (day_of_birth) then
```

Comment:
This is to deal with just calc someones age from year,
Eg: current date: 2019-04-19 person date of birth: 2009-03-20, calc age just from year gives the age 10 while its not 10 yet so -1 in case month and date are less than.